packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -781,24 +782,28 @@ export const toStyles = (
 	const nodesWithSettings = getNodesWithSettings( tree, blockSelectors );
 	const useRootPaddingAlign = tree?.settings?.useRootPaddingAwareAlignments;
 	const { contentSize, wideSize } = tree?.settings?.layout || {};
+	let ruleset = '';
+
+	if ( contentSize || wideSize ) {
+		ruleset += ':root {';
```

Comment:
Should this use `ROOT_CSS_PROPERTIES_SELECTOR` as well?

Reply:
Good spotting!